package.json

```diff
@@ -117,7 +117,7 @@
     "url": "^0.11.0",
     "uuid": "8.3.2",
     "web-vitals": "^3.0.0-rc.0",
-    "web3modal": "^1.9.8",
+    "web3modal": "^1.9.9",
```

Comment:
How about we freeze all packages at their version?

I mean - use specific version `1.9.9` and don't add prefix `^`. So, I would have:

```
  "devDependencies": {
    "@commitlint/config-conventional": "17.0.3",
    "@emotion/react": "11.10.0",
    "@emotion/styled": "11.10.0",
    "@hookform/resolvers": "2.9.7",
    "@iconify/react": "3.2.2",
    "@mui/lab": "5.0.0-alpha.95",
    "@mui/material": "5.10.0",
    ...
```

Reply:
do we have issues with it?

Reply:
It will prevent issues in the future - when some package will update it's minor version - and our app starts breaking. In my experience - it's best to update minor versions manually - after you test it thoroughly locally.

Reply:
We can think about this in the future. Never mind for now.